From 5308c86d6726283bc32bbbef7dc35acb05028512 Mon Sep 17 00:00:00 2001
From: Frode Nordahl <fnordahl@ubuntu.com>
Date: Thu, 19 Dec 2024 15:02:47 +0100
Subject: [Patch ovn 07/15] ci: Manage host/system level dependencies.
To: dev@openvswitch.org

Most dependencies are managed in the `prepare-container` job,
but there are some host/system level dependencies.

This will be used by a subsequent patch that adds system tests
that require the `vrf` kernel module to be loaded.

Signed-off-by: Frode Nordahl <fnordahl@ubuntu.com>
---
 .github/workflows/test.yml | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/.github/workflows/test.yml b/.github/workflows/test.yml
index 463d717ef..eba20aaa8 100644
--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -108,6 +108,11 @@ jobs:
         - { arch: x86, compiler: gcc, opts: --disable-ssl }
 
     steps:
+    - name: system-level-dependencies
+      run: |
+        sudo apt update
+        sudo apt -y install linux-modules-extra-$(uname -r)
+
     - name: checkout
       if: github.event_name == 'push' || github.event_name == 'pull_request'
       uses: actions/checkout@v4
-- 
2.43.0

